trie/depthFirstSync.go

```diff
@@ -252,6 +255,12 @@ func (d *depthFirstTrieSyncer) storeTrieNode(element node) error {
 	d.trieSyncStatistics.AddNumBytesReceived(uint64(numBytes))
 	d.updateStats(uint64(numBytes), element)
 
+	leafNodeElement, isLeaf := element.(*leafNode)
```

Comment:
This can be extracted in a func and used in all three syncer implementations.

Reply:
:+1: refactored